src/encoded/static/components/objectutils.js

```diff
@@ -753,6 +743,31 @@ ItemAccessories.contextTypes = {
 };
 
 
+/**
+ * Display the top section containing the breadcrumb links on summary pages.
+ */
+export const TopAccessories = ({ context, crumbs }) => {
```

Comment:
Can this be named "TopBreadCrumbSection"? "TopAccessories" seems very generic

Reply:
Intentionally generic. Conceivably, you could have a cart button without breadcrumbs.